docs/app/demo/typography/typography-demo.component.html

```diff
@@ -0,0 +1,422 @@
+<hc-subnav>
+  <h2>Typography</h2>
+</hc-subnav>
+
+<div class="demo-content">
+  <p>
+    Helper classes and examples to help with headings, body text, and more.
+  </p>
+  <h3>Fonts</h3>
+  <h4>Open Sans</h4>
+  <em>Our primary font for all things web.</em>
+  <p>
+    <a href="https://fonts.google.com/specimen/Open+Sans">https://fonts.google.com/specimen/Open+Sans</a>
+  </p>
+  <h4>Flexo</h4>
+  <em>The Health Catalyst corporate font. Only to be used for product names displayed in the header</em>
+  <img alt="cashmere app logo" src="./assets/CashmereAppLogo.svg" style="background-color:#384655; width:500px">  
```

Comment:
Could you add `margin-top: 15px` if we're doing inline styles on the logo?  Just so it doesn't run right up against the paragraph above.

Reply:
I am moving this to the style sheet...